### PR TITLE
create overview page, using routing by click on picture

### PR DESCRIPTION
**Pull request by muhammed-basaran**

### CODE REVIEW

File: components/Itemlist.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 export default function Itemlist() {
   return (
     <>
       {shoes.map((shoe) => (
-        <StyledItemListContainer>
-          <StyledImage src={shoe.img} width="50%" height="70%" />
-          <StyledItemName className="itemName">{shoe.name}</StyledItemName>
+        <StyledItemListContainer key={shoe.id}>
+          <Link href={`/../shoes/${shoe.id}`}>
+            <img src={shoe.img} max-width="200px" max-height="200px" />
```
Comment (onebarloop) [inline components/Itemlist.js:11]: Maybe think about using the `<Image />` that Next provides.

File: lib/shoes.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  {
+    id: "4",
+    producer: "Air Jordan",
+    name: "Air Jordan 6 Metallic Silver",
+    release: "02.12.2022",
+    price: "200",
+    title: "Air Jordan 6 Metallic Silver release infos",
+    description1:
+      "Der Air Jordan 6 „Metallic Silver“ greift auf ein schwarzes Nubuk-Upper zurück, während silberne Details auf den Logos und Mittelsohlen dem Sneaker eine persönliche Note verleihen. Ganz unten runden dann eisblaue Laufsohlen das Design des Sneakers ab.",
+    description2: "",
```
Comment (onebarloop) [inline lib/shoes.js:49]: Maybe think about a bette naming.

**Review by onebarloop — APPROVED** [review]
Very nice! Looking forward to order some new Air Force!

**Comment by muhammed-basaran** [comment]
> Very nice! Looking forward to order some new Air Force!

Thx, are also my fav one :)

**Pull request merged**